pallets/runtime/src/test/balances_test.rs

```diff
@@ -331,16 +330,22 @@ fn transfer_with_memo_we() {
                 alice,
                 bob,
                 200,
-                0,
                 memo_2.unwrap(),
             )),
             topics: vec![],
         },
         EventRecord {
             phase: Phase::ApplyExtrinsic(0),
-            event: EventTest::balances(BalancesRawEvent::Transfer(alice, bob, 300, 0)),
+            event: EventTest::balances(BalancesRawEvent::Transfer(alice, bob, 300)),
             topics: vec![],
         },
     ];
-    assert_eq!(System::events(), expected_events);
+    // Ignoring `frame_system` events
+    let system_events = System::events();
+    let emitted_events = vec![
+        system_events[6].clone(),
+        system_events[7].clone(),
+        system_events[8].clone(),
+    ];
+    assert_eq!(emitted_events, expected_events);
```

Comment:
It'll be better to assert only our events. No need to clone the system events.

Reply:
We are asserting our event only. That's why I take `system_events[6..8]` to assert.

Reply:
Yeah, I read that wrong. 
-not an Issue-